quilt3_local/quilt3_local/__init__.py

```diff
@@ -0,0 +1 @@
+__version__ = '0.1.0'
```

Comment:
I think this is not considered a good practice anymore, see https://www.python.org/dev/peps/pep-0396/#pep-rejection,